src/main/resources/templates/store.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/layout :: head('Assigned orders')">
+    <meta charset="UTF-8">
+    <title>Assigned orders</title>
+</head>
+<body>
+<nav th:replace="fragments/layout :: navbar(~{ :: .mr-auto}, ~{ :: .ml-auto})">
+    <ul class="navbar-nav mr-auto">
+        <li class="nav-item active">
+            <a class="nav-link" th:href="@{/}">Home</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link active" th:href="@{/store}">Store</a>
+        </li>
+    </ul>
+    <ul class="navbar-nav ml-auto">
+        <li class="nav-item">
+            <a class="nav-link" th:href="@{/logout}">Logout</a>
+        </li>
+    </ul>
+</nav>
+<style>
+    html,
+    body {
+        height: 100%;
+        background-color: whitesmoke
+    }
+
+    .flex-fill {
+        flex: 1;
+    }
+
+    .sticky-top {
+        top: 1em
+    }
+</style>
```

Comment:
Should be in a separate file, ex. `static/css/main.css`, just link it last in the `head` at it will have the highest priority.